1.test.js

```diff
@@ -0,0 +1,38 @@
+const childProcess = require('child_process')
+
+const numberGuessProgram1 = childProcess.spawn('node', ['1.js'])
+const numberGuessProgram2 = childProcess.spawn('node', ['1.js'])
+
+
+numberGuessProgram1.stdout.on('data', (output) => {
+  numberGuessProgram1.stdin.write('exit\n')
+})
+
+numberGuessProgram1.on('close', () => {
+  console.log('Program exits when \'exit\' is entered')
+})
+
+
+let currentGuess = 50
+let wonOnce = false
+
+numberGuessProgram2.stdout.on('data', output => {
+  if (output.toString().includes('Guess')) {
+    currentGuess = 50
+    numberGuessProgram2.stdin.write(`${currentGuess}\n`)
+  } else if (output.toString().includes('high')) {
```

Comment:
You could include a one time `console.log` saying that it is possible to guess again after having guessed.